lib/rake/eyes.rake

```diff
@@ -67,8 +77,12 @@ namespace :eyes do
     HipChat.log "#{Emoji.find_by_alias('muscle').raw}  Force merging baselines #{args}"
     force_merge_eyes_baselines(args[:branch], args[:base])
   end
+  task :copy, [:branch, :base] do |_, args|
+    HipChat.log "#{Emoji.find_by_alias('clipboard').raw}  Copying baselines #{args}"
```

Comment:
It'd sure be nice if `gemoji` came with an `emojify` helper like they [suggest in their README](https://github.com/github/gemoji#example-rails-helper).  Maybe we should add one here so you can write

```ruby
HipChat.log emojify ":clipboard:  Copying baselines #{args}"
```